docs/example-terraform.md

```diff
@@ -0,0 +1,187 @@
+# Terraform example
+
+We will be looking at how to use Kapitan to compile terraform files with Jsonnet as the input type. It's possible to use other input types, however, Jsonnet is recommended. 
+For example, we could use the Kadet input to generate terraform files but this would require templates to be written in YAML then to rendered into JSON.
```

Comment:
"then rendered into JSON" *

Reply:
This is actually not true. Kadet will work with both JSON or YAML.

Reply:
Thanks for spotting that. 

@ramaro Thanks that sounds important. It means integration would be much simpler since we can convert HCL to JSON. Kadet can then use the pre-existing JSON as  input templates.